### PR TITLE
Typescript migration stub 

### PR DESCRIPTION
**Pull request by anggiaj**
add typings on typescript migration stub

### CODE REVIEW

File: src/migrate/stub/ts.stub [hunk 1]
```diff
@@ -1,15 +1,15 @@
 import * as Knex from "knex";
 
-exports.up = function (knex: Knex): Promise<any> {
+export async function up(knex: Knex): Promise<Knex.SchemaBuilder> {
```
Comment (elhigu) [inline src/migrate/stub/ts.stub]: I don't thing tat typing is right. AFAIK Migration does not always resolve with Knex.SchemaBuilder instance.

Comment (anggiaj) [reply src/migrate/stub/ts.stub]: true, isn't always return `Knex.SchemaBuilder`, but the default template use `createTable` and `dropTable` which both returns `Knex.SchemaBuilder`

Comment (anggiaj) [reply src/migrate/stub/ts.stub]: updated

**Comment by elhigu** [comment]
thanks, now it looks good :) gotta wait for tests to finish though.

**Pull request merged**